evmcore/tx_pool.go

```diff
@@ -185,6 +185,8 @@ type TxPoolConfig struct {
 	GlobalQueue  uint64 // Maximum number of non-executable transaction slots for all accounts
 
 	Lifetime time.Duration // Maximum amount of time non-executable transaction are queued
+
+	DisableTxPoolValidation bool
```

Comment:
Please document this option. In particular, mention that this is only to be used for integration tests.